lib/authentication/auth_okta.js

```diff
@@ -100,7 +91,7 @@ function AuthOkta(password, region, account, clientType, clientVersion, httpClie
   *
   * @returns {Object}
   */
-  function step1(authenticator, serviceName, account, username) {
+  async function step1(authenticator, serviceName, account, username) {
```

Comment:
I know you didn't write this but does the function parameters actually get used in this function? Seems like they can be removed

Reply:
Yes, the driver is using them in the step1 function.

`   let header;
    if (serviceName) {
      header = {
        'HTTP_HEADER_SERVICE_NAME': **serviceName**
      };
    }

    // JSON body to send with POST request
    const body = {
      'data': {
        'ACCOUNT_NAME': **account**,
        'LOGIN_NAME': **username**,
        'PORT': port,
        'PROTOCOL': protocol,
        'AUTHENTICATOR': **authenticator**,
        'CLIENT_APP_ID': clientAppId,
        'CLIENT_APP_VERSION': clientAppVersion
      }
    };

    // POST request to get SSO URL and token URL
    return await httpClient.post(url, body, {
      headers: header
    });
  }`

Reply:
My bad, I thought I was looking at the entire function hehe